cmake/common.cmake

```diff
@@ -4,6 +4,11 @@
 set(CMAKE_MODULE_PATH "${CCF_DIR}/cmake;${CMAKE_MODULE_PATH}")
 
 set(MSGPACK_INCLUDE_DIR ${CCF_DIR}/3rdparty/msgpack-c)
+set(FLATBUFFERS_INCLUDE_DIR ${CCF_DIR}/3rdparty/flatbuffers/include)
+
+execute_process(
+    COMMAND flatc --cpp ${CCF_DIR}/src/flatbuffers/data.fbs
```

Comment:
data is awfully generic, I suggest kv_frame instead. Alternatively src/kv/frame.fbs if we want to keep the idl files close to the source.